src/test/java/com/courtcanva/ccfranchise/services/StaffServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package com.courtcanva.ccfranchise.services;
+
+import com.courtcanva.ccfranchise.constants.AUState;
+import com.courtcanva.ccfranchise.dtos.StaffGetDto;
+import com.courtcanva.ccfranchise.dtos.StaffPostDto;
+import com.courtcanva.ccfranchise.mappers.StaffMapper;
+import com.courtcanva.ccfranchise.models.Franchisee;
+import com.courtcanva.ccfranchise.models.Staff;
+import com.courtcanva.ccfranchise.repositories.StaffRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class StaffServiceTest {
+
+    @Mock
+    private StaffRepository staffRepository;
+
+    @Mock
+    private StaffMapper staffMapper;
```

Comment:
We don't need to mock mappers, just use real instances instead.

Reply:
thx, already fix it